### DISCUSSION TITLE
Can we use GitHub discussions as a mailing list?

### DISCUSSION THREAD

**Discussion by dahavey** [microsoft/ebpf-for-windows#399] [Q&A] [answered]
Does GitHub discussions allow mailing listserv functions. In other words will it email the group and can we send email to it?

**Comment by dthaler** [comment]
Testing a reply all from email.

From: Daniel M. Havey ***@***.***>
Sent: Thursday, August 19, 2021 4:22 PM
To: microsoft/ebpf-for-windows ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [microsoft/ebpf-for-windows] Can we use GitHub discussions as a mailing list? (#399)


Does GitHub discussions allow mailing listserv functions. In other words will it email the group and can we send email to it?

-
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Febpf-for-windows%2Fdiscussions%2F399&data=04%7C01%7Cdthaler%40microsoft.com%7Cbd3fec5f8dbf46da89d608d963682841%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637650121295697053%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=bS%2BD%2BCz48xM64R3VmD%2Bay0zUHs96kJGlaS9sZuZR%2FFc%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABR6SSBQHWJMNY2TDESZA7LT5WGZZANCNFSM5CPGL6NA&data=04%7C01%7Cdthaler%40microsoft.com%7Cbd3fec5f8dbf46da89d608d963682841%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637650121295707013%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=hDlcZG1qCykST3h%2FMQZ0pKR6VCdNJK%2BEX3Tly5YaM2A%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cdthaler%40microsoft.com%7Cbd3fec5f8dbf46da89d608d963682841%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637650121295707013%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=xJ8vxhhBEygM11QvjZFOVv67RF4EFRMWDD5nghVjaiY%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26utm_campaign%3Dnotification-email&data=04%7C01%7Cdthaler%40microsoft.com%7Cbd3fec5f8dbf46da89d608d963682841%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637650121295716968%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=nTedVO%2FsCoP9U05JIlIoNzRP0E5jAYU2AT1q6j2mUmM%3D&reserved=0>.

**Comment by dthaler** [comment, marked as answer]
A: Yes.  You can subscribe to discussion notifications by clicking "Watch" (or "Unwatch") and selecting "Custom" -> "Discussions" (or by selecting "All Activity" if you want to receive notifications about everything else too).
